static/js/src/advantage/manage-subscription.js

```diff
@@ -1,93 +1,252 @@
-import { resizeContract, cancelContract } from "./contracts-api.js";
+import {
+  resizeContract,
+  cancelContract,
+  getPurchase,
+} from "./contracts-api.js";
+
+const stripe = window.Stripe(window.stripePublishableKey);
+
+const getMessage = (code, default_message) => {
+  const map = {
+    resizing_machines_success: "Resizing successful! Reloading page...",
+    cancelling_subscription_success:
+      "Subscription cancelled! Reloading page...",
+    resizing_machines_fail:
+      "<strong>Payment method:</strong> There was an error with the payment please <a href='/advantage/payment-methods'>update your payment methods</a> to fix it.",
+    subscription_missing:
+      "<strong>Cancelling subscription failed:</strong> It could be that you have a pending payment that is blocking this action. Contact <a class='p-notification__action' href='https://ubuntu.com/contact-us'>Canonical sales</a> if the problem persists.",
```

Comment:
```suggestion
      "<strong>Could not cancel subscription:</strong> It could be that you have a pending payment that is blocking this action. Contact <a class='p-notification__action' href='https://ubuntu.com/contact-us'>Canonical sales</a> if the problem persists.",
```